### ISSUE TITLE
Would it be possible to update the dependency to stylelint-order?

### ISSUE THREAD

**Issue by julienw** [ream88/stylelint-config-idiomatic-order#66]
Hey,
I noticed that this package depends on a quite old version of `stylelint-order`, which transitively depends on outdated versions of `postcss` that are flagged as having security issues.
Would it be possible to update this dependency?
Thanks!

**Comment by jd-0001** [comment]
@ream88 stylelint-order's older version causing postcss version mismatch resulting in below error:
```
postcss-sorting: postcss.plugin was deprecated.
```

Can you please upgrade it to v5

**Comment by julienw** [comment]
> Hey, I noticed that this package depends on a quite old version of `stylelint-order`, which transitively depends on outdated versions of `postcss` that are flagged as having security issues. Would it be possible to update this dependency? Thanks!

postcss v7 was updated so it's now possible to fix the security issues (at least with yarn).

That said I'm considering importing the logic for this package directly to our project so that we can track `stylelint-order` directly instead.

**Comment by ream88** [comment]
Updated 😊 and released `9.0.0`

**Comment by julienw** [comment]
Thanks! Could you push the tag to github as well ? :-)
